app/controllers/metrics_controller.rb

```diff
@@ -0,0 +1,17 @@
+class MetricsController < ApplicationController
+  def index
+    @hash = { "labels" => [],
+      "datasets" =>[
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Surrounding space missing for operator '=>'.